client/app/components/project/package-section.js

```diff
@@ -21,6 +21,16 @@ export default class ProjectPackageSectionComponent extends Component {
     return payablePackage;
   }
 
+  get invoices() {
```

Comment:
Ooh okay, so if I'm following correctly -- you're trying to create an array of alll the invoices from every package regardless of package type -- can you explain why the reduce necessary here?

Reply:
The reducer here is a common array flattener — if I map out the invoices of packages, I get an array of arrays. I need to flatten it out so I can treat them as a one dimensional array.